server/cli/src/cli.rs

```diff
@@ -624,6 +643,36 @@ async fn main() -> anyhow::Result<()> {
                 .arg(generated_file_path.clone())
                 .status()
                 .expect(&format!("failed to open file {:?}", generated_file_path));
+        },
+        Action::ToggleReport { code, is_custom, enable, disable } => {
+            let connection_manager = get_storage_connection_manager(&settings.database);
+            let con = connection_manager.connection()?;
+
+            let mut filter = ReportFilter::new().code(EqualFilter::equal_to(&code));
+            match is_custom {
+                Some(value) => {
+                    filter = filter.is_custom(value);
+                },
+                None => {}
+            }
```

Comment:
This is great thanks Ethan.

One thing I didn't explicitly state in the issue was being able to disable the standard reports while maintaining custom reports. (This is in the case where we have custom reports replacing standard reports for a client, but the custom reports are on a more recent version of OMS than some of their remote sites. The remote sites in this case would show the standard code-equivalent report where the desired behaviour might be to show no reports).

And you've written this in a generic way so we can have a targeted response so thank you for thinking of all cases.